### PR TITLE
Updated new details of SN2

### PR DESCRIPTION
**Pull request by gagichce**
Hi OTF!

Updated the details of SN2 to match it's new purpose.

Happy to make any changes!

### CODE REVIEW

File: docs/subnet-pages/subnet-2.md [hunk 1]
```diff
@@ -6,29 +6,25 @@ title: "Subnet 2"
...
 
-BitTranslate’s process is a synergy between validators and miners. Validators begin by proceeding organic requests or by extracting sample texts in specific languages from the dataset. To guarantee uniqueness, these texts are processed using a Text Generation model.
-
-Once processed, requests are distributed to a network of specialized subnet miners, which are ready for translation. Using advanced AI models, miners ensure the translations are both accurate and of high quality. Validators then step in again to rigorously evaluate these translations using multiple quality estimation techniques:
-
-- **BertScore**: Prioritizing the essence of content, this model rewards translations that resonate deeply with the original material.
-- **TransQuest Technology**: Beyond just words, TransQuest evaluates the depth and context of translations, ensuring they capture the full spectrum of meanings from the source.
-
-Then, miners are rewarded with TAO tokens based on their performance. The mechanisms ensures competitiveness and hence continuous excellence in BitTranslate.
+The purpose of the Omron subnet is to provide verified AI inferences to optimize staking and re-staking yields.
```
Comment (rajkaramchedu) [inline docs/subnet-pages/subnet-2.md:16]: Please add a new line here (just insert a blank line) so it breaks up the paragraph.

File: docs/subnet-pages/subnet-2.md [hunk 2]
```diff
@@ -6,29 +6,25 @@ title: "Subnet 2"
...
+Omron allows a variety of yield optimization models to be hosted and run against the network while being zk-verified.
+Specifically, this subnet enhances the Bittensor network as a whole by providing external uses for protocols outside the Bittensor ecosystem; thereby amplifying bittensor’s mission throughout a variety of external networks.
```
Comment (rajkaramchedu) [inline docs/subnet-pages/subnet-2.md:18]: Is this what you mean? See below. If correct, use this below wording:

"In specific, this subnet enhances the Bittensor network by providing utility to protocols and networks outside the Bittensor ecosystem. As a result, this subnet amplifies Bittensor’s mission throughout a variety of such external protocols and networks."

File: docs/subnet-pages/subnet-2.md [hunk 3]
```diff
@@ -6,29 +6,25 @@ title: "Subnet 2"
...
+As well, this subnet provides internal uses for guiding bittensor users towards the most profitable validators to delegate to which benefits delegators and thereby the network as a whole.
```
Comment (rajkaramchedu) [inline docs/subnet-pages/subnet-2.md:19]: Please use this wording below:

"Furthermore, this subnet provides internal benefits by guiding Bittensor users towards the most profitable validators to delegate to. This, in turn, benefits delegators and thereby benefits the Bittensor network as a whole."

**Review by rajkaramchedu — CHANGES_REQUESTED** [review]
Good work! See my comments. Thanks!

**Comment by rajkaramchedu** [comment]
Redid these changes myself, so we can update the subnet page quickly. Closing this PR. See https://github.com/opentensor/developer-docs/pull/201

**Comment by gagichce** [comment]
Thanks for merging this in!

**Pull request merged**